### PR TITLE
Edge Node Resource Consumption SLO changes on performance

### LINKED ISSUE #630: Enahance the Performance framework for Benchmark the Resource Consumption SLO's
**What would you like to be added**:
Enhance the performance framework for Benchmarking Resource consumtion SLO's

**Why is this needed**:
This is helps in measuring the resource consumption(Memory and CPU) of a KubeEdge Node when user deploy various no.of application Pod's.Which will help in validating the KubeEdge node performance based on application load.

### PR DESCRIPTION
**Pull request by pavan187**
**What type of PR is this?**

/kind test

**What this PR does / why we need it**:
This PR will banchmark the Resource consumption (Memory and CPU) for perticular Edge node under various load conditions.

**Which issue(s) this PR fixes**:
Fixes #630

**Special notes for your reviewer**:
/cc @edisonxiang @rohitsardesai83 @sids-b

### CODE REVIEW

**Comment by sids-b** [comment]
ping @edisonxiang @rohitsardesai83 for review

File: tests/performance/common/common.go [hunk 1]
```diff
@@ -46,8 +46,44 @@ const (
...
+	ContainerStats    = "/containers/%s/stats?stream=0" //stream – 1/True/true or 0/False/false, pull stats once then disconnect. Default true.
 )
 
+type ContainerList []struct {
+	ID     string   `json:"Id"`
+	Names  []string `json:"Names"`
+	Labels struct {
+		IoKubernetesContainerName string `json:"io.kubernetes.container.name"`
+		IoKubernetesPodName       string `json:"io.kubernetes.pod.name"`
+	} `json:"Labels"`
```
Comment (fisherxu) [inline tests/performance/common/common.go]: Please remove this struct defination out of ContainerList, used as a field instead.

File: tests/performance/common/common.go [hunk 2]
```diff
@@ -46,8 +46,44 @@ const (
...
+		} `json:"cpu_usage"`
+		SystemCPUUsage uint64 `json:"system_cpu_usage"`
+	} `json:"precpu_stats"`
+	MemoryStats struct {
+		Usage uint64 `json:"usage"`
+		Limit uint64 `json:"limit"`
+	} `json:"memory_stats"`
+	Name string `json:"name"`
+	ID   string `json:"id"`
+}
```
Comment (fisherxu) [inline tests/performance/common/common.go]: ditto

File: tests/e2e/utils/config.go [hunk 3]
```diff
@@ -44,6 +44,7 @@ type Config struct {
 	Namespace                      string            `json:"namespace"`
 	ControllerStubPort             int               `json:"controllerstubport"`
 	Protocol                       string            `json:"protocol"`
+	DockerClinet                   string            `json:"dockerclient"`
```
Comment (fisherxu) [inline tests/e2e/utils/config.go]: where init the it? :)  and s/DockerClinet/DockerClient/g

**Comment by kevin-wangzefeng** [comment]
/close
We can always reopen the PR if necessary :-)

**Pull request closed without merging**